src/channel.rs

```diff
@@ -733,3 +753,23 @@ impl Channel {
         &self.cq
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::sync::Arc;
+    use crate::env::Environment;
+    use crate::ChannelBuilder;
+
+    #[test]
+    fn test_grpc_min_message_size_to_compress() {
```

Comment:
I think they need to be guarded by `nightly` feature.

Reply:
done